pages/ext/discord.vue

```diff
@@ -181,6 +198,14 @@ export default {
         Banner,
         ExtLink,
     },
+    async asyncData({ store, query, redirect }) {
+        const token = query.token
+        if (!token) {
+            redirect('/')
```

Comment:
can we also show the warning text (`invalidAttendee`) on the page instead of redirecting?

Reply:
This redirection will only occur when no `token` query appeared in URL like `{host}/pycontw-2021/zh-hant/ext/discord`. The site will show warning text if given a invalid token. Do you mean we should also make a popup window to notify users that he/she should enter a valid token in URL?

Reply:
nah, text on page (same as invalid token) would be fine

Reply:
Got it, I've updated it. Please have a look. Thanks!